site/en/hub/overview.md

```diff
@@ -14,7 +14,7 @@ code.
 ```python
 import tensorflow_hub as hub
 
-model = hub.KerasLayer("https://tfhub.dev/google/nnlm-en-dim128/2")
+model = hub.KerasLayer("https://www.kaggle.com/models/google/nnlm/TensorFlow2/en-dim128/2")
```

Comment:
Note: I based this on the example in https://www.kaggle.com/models/google/nnlm but did not confirm whether this specific code works.